doc/index.rst

```diff
@@ -263,9 +263,11 @@ ABI checks
 
 Generally speaking, the `ABI <https://en.wikipedia.org/wiki/Application_binary_interface>`_ of a library might break for various reasons. A detailed explanation and a list of DOs and DON'Ts can be found in the `KDE Community Wiki <https://community.kde.org/Policies/Binary_Compatibility_Issues_With_C%2B%2B>`_.
 
-The ABI checks with `industrial_ci` can be enabled by setting 'ABICHECK_URL' to a code archive with your stable version (e.g. tagged version or older tagged versions of your package)
+The ABI checks with `industrial_ci` can be enabled by setting 'ABICHECK_URL' to the stable version (i.e. you want to check against with this for sure) of your code. Either the package's branch or the tagged version can be passed. Pointing to a branch is the simplest thus recommended but it needs to be stable, so it is really up to your discretion.
+
 The following is a few examples of those URL:
 
+  - https://github.com/ros-industrial/ros_canopen/tree/kinetic
```

Comment:
No, you can't point it to a branch. You have to point it to an archive. Luckily, github & co provide archive links for branches ( and tags and commits).